contents/blog/company-survey-h1-2024.md

```diff
@@ -0,0 +1,99 @@
+---
+date: 2024-06-25
+title: What employees really think about PostHog
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author:
+  - james-temperton
+featuredImage: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/mykonoshackathon_979cf7bec6.jpeg
+featuredImageType: full
+category: Inside PostHog
+tags:
+  - Culture
+---
+
+At PostHog, we want to build the [most transparent company in the world](/handbook/people/overview) – and the best place for people to work in general. To find out how we’re doing, we run bi-annual team surveys. And to be totally transparent, we’re going to publish them here for anyone to see.
+
+We had a 79% response rate, or 33 out of 42 eligible respondents. For our last survey, we had a 74% response rate. Since then, we’ve added 14 new team members and lost three. Ideally we’d get a response rate of 90% or higher, though we did have a good cross section of teams and tenures.
+
+We asked 14 questions based on the ones used by [Culture Amp](https://www.cultureamp.com/). Questions cover categories such as company, confidence, culture, and growth. 12 of the questions we asked required a response ranging from 1, meaning strongly disagree, to 5, meaning strongly agree:
+
+* PostHog is in a position to really succeed over the next three years.
+* I understand PostHog's goals and can see how my work contributes to them.
+* At PostHog, we have open and honest two-way communication.
+* I receive appropriate recognition for my work at PostHog.
+* I believe that my total compensation (salary + equity + benefits) is fair, relative to similar roles at other companies.
+* My manager and team around me genuinely care about my wellbeing.
+* The leaders at PostHog keep people informed about what is happening.
+* I feel like I am learning and growing at PostHog.
+* Generally, I believe my workload is reasonable for my role and I am able to arrange time out from work when I need it.
+* The support I am receiving and processes we have in place allow me to do my best possible work.
+* I would recommend PostHog as a great place to work.
+* I see myself still working at PostHog in two years' time.
+
+And two of which were open text fields:
```

Comment:
- These questions are mandatory
- They are particularly useful and often the most acted-upon